test/tc_basic.rb

```diff
@@ -114,7 +114,7 @@ def test_set_and_get_point_from_wkt
               obj_ = klass_.new
               assert_nil(obj_.latlon)
               obj_.latlon = 'SRID=1000;POINT(1 2)'
-              assert_equal(@factory.point(1, 2), obj_.latlon)
+              assert_equal(RGeo::Cartesian.preferred_factory(srid: 1000).point(1, 2), obj_.latlon)
```

Comment:
@dschweisguth if this change is about supporting rgeo 0.4 then I'd skip it. It seems that all our tests pass with rgeo 0.6.

Projects that depend on rgeo-activerecord branch 1.0 (like ours) can use rgeo 0.6 most probably, as rgeo-activerecord 1.x has a dependency on all rgeo 0.x versions:

```
  spec.add_dependency 'rgeo', '~> 0.3'
```

So, I'd vote for removing this change from the test suite.